test/cmdlineTests/yul_leading_or_trailing_dot_in_name/args

```diff
@@ -0,0 +1 @@
+--strict-assembly --optimize
```

Comment:
You could `--debug-info none` to remove the unnecessary debug comments from the output.

```suggestion
--strict-assembly --optimize --debug-info none
```

But I agree with @nikola-matic, a Yul syntax test is enough if you can repr

Also, the original issue says that `--optimize` is not necessary to trigger this. I suspect this is simply due to #13972, i.e. we run some minimal optimizations even when the optimizer is technically disabled. So it probably still makes sense to have it here to make it more explicit.

Reply:
I could reproduce with a yul objectCompiler test, interestingly syntax test was not enough